Project-Lifecycle.md

```diff
@@ -98,10 +98,26 @@ vote to abstain from representation on the TSC.
 
 ## Applying to join
 
+For a project to be accepted as a top-level project under the Node.js
+Foundation it must help better the Node.js ecosystem.  It is important
+that proposals include a clear explanation of why this will be the
+case.  A few areas you may want to include:
+
+* how will the project help the Node.js ecosystem grow as a whole ?
+* what set of Node.js users is the project interesting/critical to ?
+* is there a direct relationship between the project and core Node.js
+  or one of the existing top-level projects?  For example is it a
+  Node.js dependency ?
```

Comment:
I wonder if "is it a Node.js dependency" might be misinterpreted. Suggestion "does Node.js itself depend on it"